Gemfile

```diff
@@ -147,6 +147,7 @@ def override_gem(name, *args)
     dependencies.delete(dependency)
 
     gem name, *args
+    Bundler::UI::Shell.new.warn "Caution, using override_gem for #{name} with args: #{args.inspect}, review Gemfile.dev.rb if this is not desired!"
```

Comment:
Yeah, wording is hard.  I'll wait to see what others think and make any suggestions.